bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -70,9 +74,15 @@ In this case:
 
 ### Panel Layout
 
-Unfortunately it is not easy to find out which panel gets which id, and this becomes pretty important if you have lots of them and want to assign rules. 
+If you want to program individual panels, it can be hard to figure out which panel has which ID. To make this easier, there is Layout channel on the Nanoleaf controller thing in OpenHAB. 
```

Comment:
```suggestion
If you want to program individual panels, it can be hard to figure out which panel has which ID. To make this easier, there is Layout channel on the Nanoleaf controller thing in openHAB. 
```